api/proto/config/storage.proto

```diff
@@ -131,6 +131,14 @@ message ContentTree {
   uint64 maxSizeBytes = 6;
   SignatureInfo siginfo = 7;
   string displayName = 8; // Optional friendly name echo'ed in info
+
+  // For container ContentTree, sha256 might not be available in config.
```

Comment:
While I'm not against this improvement -- I must point out that I'm moving closer to rethinking what SHAs do we use where (that's part of the discussion we're having with @deitch and some upstream containerd folks). Expect a proposal sometime soon that may streamline this area.

Reply:
I guess this is mostly a comment for @eriknordmark and @adarsh-zededa